app/layout.tsx

```diff
@@ -7,7 +7,10 @@ import { Navbar, Footer, ScrollToTop } from "@/components";
 const inter = Inter({ subsets: ["latin"] });
 
```

Comment:
We can use these types to ensure type safety when working with the metadata object
```
type Metadata = {
  title: TitleMetadata;
  description: string;
};

type TitleMetadata = {
  default: string;
  template: string;
};

```